Gruppe.A1.End-to-End_Encryption_mit_IPv6_und_integriertem_IPSec.ThushjandanPonnudurai.PascalBaumann.Termpaper.tex

```diff
@@ -111,19 +112,104 @@ \subsection{IPsec}
 
 The next consideration concerns the key management. Although manual key management can be used, it's generally not advised and disables the replay detection mechanism.The most widely used key management is the Internet Key Exchange (IKE), which is available in its original form (IKEv1) and in a new, simpler version (IKEv2). In network environment where mainly Microsoft systems are used, the usage of Kerberised Internet Negotiation of Keys (KINK) is available, as this depends on the Kerberos system. \parencite[4-5]{Bellovin2009}
 
-\section{IPsec}
-\label{sec:IPsec}
+\section{ISAKMP}
+\label{sec:ISAKMP}
 
-\subsection{Algorithmen \& DHs}
+\subsection{Algorithms \& DHs}
 \label{ssec:AlgoDH}
+Data confidentiality is the key requirement for any VPN. The confidentiality of the data is provided by the encryption algorithm. That's why it is very important to choose the optimal encryption algorithm. \parencite{Snader2006}
 
-\subsection{Phase 1}
+There are two categories of cryptography algorithms:
+\begin{itemize}
+	\item Symmetric.
+	\item Asymmetric
+\end{itemize}
+
+\subsubsection{Symmetric}
+\label{sssec:symmetric}
+Sender and receiver are using the same secret key to encrypt and to decrypt the messages. The main problem with these categories of algorithms is finding a way to distribute the key without anyone else finding it.
+
+Some symmetrical algorithms:
+\begin{itemize}
+	\item AES (128, 192 or 256 bit)
+	\item DES (56-bit)
+	\item Triple-DES or 3DES (3x 56-bit = 168-bit).
+	\item Blowfish (128, 192 or 256 bit)
+	\item CAST (128-bit)
+	\item Camellia (128, 192 or 256 bit)
+\end{itemize}
+AES-256 is commonly used in VPNs. DES is not secure anymore, because a DES key can be cracked in less than 24 hours. \parencite{Bollapragada2005}
+
+\subsubsection{Asymmetric}
+\label{sssec:asymmetric}
+Asymmetrical algorithms, also known as public key algorithm, use separate keys for encryption and decryption. The public key is used for the encryption and can be made public. The private key is used for decryption and must be kept secret. The public key and private key are mathematically related, but it is not possible to find out the private key from the public key. 
```

Comment:
Please add what is costly with Pub/PriKey Encryption, e.g. why it's secure in more detail than just that they are related. Remember, our readers are from the field